untitled-sdl-tank-game/utils/utils.cpp

```diff
@@ -0,0 +1,18 @@
+#include "utils.h"
+#include "math.h"
+
+int distanceFromPointToLine(Line line, Point point)
```

Comment:
Optionally (may be in different pull request) please add  some dummy unit tests by:
* adding dependent [project](https://github.com/pietrik123/untitled-sdl-tank-game/wiki/Add-a-project-to-a-Solution-in-MS-Visual) to repo (like [this)](https://github.com/pietrik123/untitled-sdl-tank-game/commit/9d5a40b80e7a3a4fc74982d723bea499bb6f6030#diff-900bcabb885a4243a6ce2c75b247bcec)
* adding *.cpp test file ([example](https://github.com/pietrik123/untitled-sdl-tank-game/blob/9d5a40b80e7a3a4fc74982d723bea499bb6f6030/test_dep_proj2/dummy_test.cpp))